lib/matplotlib/path.py

```diff
@@ -822,45 +822,48 @@ def circle(cls, center=(0., 0.), radius=1., readonly=False):
           Bezier Cubic Splines <https://www.tinaja.com/glib/ellipse4.pdf>`_.
         """
         MAGIC = 0.2652031
-        SQRTHALF = np.sqrt(0.5)
+        SQRTHALF = (1 / 2) ** (1 / 2)
         MAGIC45 = SQRTHALF * MAGIC
+        MAGICSUM = SQRTHALF + MAGIC45
+        MAGICDIFF = SQRTHALF - MAGIC45
 
-        vertices = np.array([[0.0, -1.0],
```

Comment:
I guess I would just write this is as `vertices = np.array([...])` without even bothering with the `dtype=float` which is unneeded.